garrysmod/gamemodes/terrortown/gamemode/lang/french.lua

```diff
@@ -1019,7 +1019,7 @@ L.aw_brn2_title = "PyroÏde"
 L.aw_brn2_text  = "a été entendu rire aux éclats après avoir brûlé un paquet de ses victimes."
 
 L.aw_brn3_title = "BrÛleur Pyrrhique"

```

Comment:
Why all those uppercase letters with accent ?
Is it because of string.upper bug with utf8 ?

If yes i'm pretty sure there is a working utf8 function

Reply:
Yes. string.upper doesn't effect letters with accent

Reply:
I took a look at the utf8 lib, it doesn't include any upper func/method.

We could maybe use this 

https://github.com/NebulousCloud/helix/blob/e09e6c37c79dd95d6d53842e47571b4c75b2a78b/gamemode/core/libs/thirdparty/sh_utf8.lua

https://github.com/NebulousCloud/helix/blob/e09e6c37c79dd95d6d53842e47571b4c75b2a78b/gamemode/core/libs/thirdparty/data/sh_utf8_casemap.lua

Reply:
Hmm...